Slightly Revamps the Departmental Shuttle

Onyxarias opened the pull request:
<!-- By ticking or leaving ticked the option "Allow edits and access to secrets by maintainers" you give permission for repository maintainers to push changes to your branch without explicitly asking. -->

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do

This PR slightly revamps the Departmental emergency shuttle. The changes are mainly focused in the department rooms, making them more themed and interesting and reducing the number of seats; they've never been filled, so having more space for other things is better. 

Some things are added as well, including some emergency supplies, as well as a mech charging station in the science bay. The rest of the shuttle had a small aesthetic overhaul, with some floor swaps and slight shifting. 

<!-- Include a small to medium description of what your PR changes. -->
<!-- Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
<!-- If your PR fixes an issue, add "Fixes #1234" somewhere in the PR description. This will automatically close the bug upon PR submission. -->

## Why It's Good For The Game

This was done mainly on the suggestions of others, who found the shuttle to be lacking, and the departmental rooms to be an eyesore. This should improve it visually, plus add some supplies that can be found on some of the other shuttles.

<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Images of changes

![image](https://github.com/user-attachments/assets/fd31812f-4c08-4c29-91c0-608c1c8d2d67)
![image](https://github.com/user-attachments/assets/95f7e72f-f62e-43c7-8334-70cdbd0b3db2)


<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->

## Testing

Spun up a server, bought and called the shuttle. Had to tick a box to keep runtimes from bricking the game. Shuttle flew in fine, and everything worked. 

<!-- How did you test the PR, if at all? -->

<hr>

### Declaration

- [ x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  <!-- Replace the box with [x] to mark as complete. -->
  <!-- Ensure there are no spaces between the x and the square brackets [] else this will not work properly. -->
  <hr>

## Changelog

:cl:
add: Added new tools and equipment to the Departmental shuttle
tweak: Changed the overall look of some parts of the shuttle, shifted the bar up one tile
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

warriorstar-orion reviewed (changes requested):
The purple/white decals cutting off here at uneven points feels awkward:

![2024_11_01__02_07_58__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/d25d7b93-0c43-471d-8f2d-7c0430266b10)

This plant seems out of place:

![2024_11_01__02_08_04__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/896ce4d3-7adf-451f-b4e1-08819997f812)

Could probably remove these decals out from every chair.

![2024_11_01__02_08_23__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/b443c9e8-0f0a-4e83-a1f6-ff42fd149184)

Light on the window:

![2024_11_01__02_10_34__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/8e28d0b9-ba77-4bbd-819b-c1c53d06f485)

I personally don't like when these two sets of floor tiles meet flush without a border so I'd recommend something like this but up to you:

![2024_11_01__02_07_50__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/92d07cda-9c26-406e-8bd1-676b4fdc80fd)

I'd flip these yellow tiles on the edges of the eng cabin around but also up to you:

![2024_11_01__02_10_02__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/256a2a8b-ea25-4996-ba55-3aa0ceb15ac9)


Maybe even use the yellow wood siding:

![2024_11_01__02_18_49__paradise dme  emergency_dept dmm  - StrongDMM](https://github.com/user-attachments/assets/6971c40a-a72a-4a97-b7e7-ed1fb7144524)

Onyxarias commented:
> The purple/white decals cutting off here at uneven points feels awkward:
> 
Definitely, I noticed it too. Should blend in better now
![image](https://github.com/user-attachments/assets/44b14756-88c7-4904-8f2a-cc2af7d726a7)

> This plant seems out of place:
> 
>
Yeah, fair. Removed it.

> Could probably remove these decals out from every chair.
> 
Agreed. I updated it though to match the style now present in the engineering room and bridge,
![image](https://github.com/user-attachments/assets/eef4a5bd-8268-4631-a3fc-f907c275faf0)

> 
> Light on the window:
> 
Lmao. Wasn't me but I didn't even catch that. Fixed.

> I personally don't like when these two sets of floor tiles meet flush without a border so I'd recommend something like this but up to you:
> 
Honestly, looking at it now, I agree. Dunno if I want to use the siding too much in the future, but here it looks good.
![image](https://github.com/user-attachments/assets/20a0e297-a051-4736-9115-5c234bdbbeb7)

> I'd flip these yellow tiles on the edges of the eng cabin around but also up to you:
> 
> Maybe even use the yellow wood siding:
> 
Agreed on both counts, with a second look. Flipped and with the siding, it honestly looks pretty great. Changed.
![image](https://github.com/user-attachments/assets/5736648a-2e3d-4ce7-93f2-28fcd378981d)
